com.microsoft.mrtk.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -181,26 +189,31 @@ public override void ProcessInteractor(XRInteractionUpdateOrder.UpdatePhase upda
                     {
                         isRelaxedBeforeSelect = false;
                     }
+                }
+            }
+        }
 
-                    // Use Pose Sources to calculate the interactor's pose and the attach transform's position
-                    if (AimPoseSource != null && AimPoseSource.TryGetPose(out Pose aimPose))
-                    {
-                        transform.SetPositionAndRotation(aimPose.position, aimPose.rotation);
-
-                        if (hasSelection)
-                        {
-                            float distanceRatio = PoseUtilities.GetDistanceToBody(aimPose) / refDistance;
-                            attachTransform.localPosition = new Vector3(initialLocalAttach.position.x, initialLocalAttach.position.y, initialLocalAttach.position.z * distanceRatio);
-                        }
-                    }
+        public void Update()
```

Comment:
nit: Unity methods can/should be non-public

Reply:
Done